redash/query_runner/pg.py

```diff
@@ -327,7 +327,8 @@ def _get_tables(self, schema):
                             column_name,
                             ordinal_position AS pos
             FROM svv_columns
-            WHERE table_schema NOT IN ('pg_internal','pg_catalog','information_schema')
+            WHERE table_schema NOT IN ('information_schema')
+            AND table_schema NOT LIKE 'pg_%'
```

Comment:
While not very likely it is possible someone might name their schema with a `pg_` prefix (maybe when [P&G](https://us.pg.com/) adopts Redshift). How about we keep the original list and limit the `LIKE` prefix to `pg_temp_`?

Reply:
good point, sure will modify